packages/@aws-cdk/aws-cognito/lib/user-pool-email.ts

```diff
@@ -56,6 +56,13 @@ export interface UserPoolSESOptions {
    * @default - The same region as the Cognito UserPool
    */
   readonly sesRegion?: string;
+
+  /**
+   * SES Verified custom domain to be used to verify the identity
+   *
+   * @default - no domain
```

Comment:
should be

```
@default no domain
```

(I know the others in this file are written with the `-`, but they shouldn't be. You can remove the `-` from the others here as well if you want to).